generator/beat/{beat}/Makefile

```diff
@@ -23,21 +23,18 @@ copy-vendor:
 	rm -rf vendor/github.com/elastic/beats/.git vendor/github.com/elastic/beats/x-pack
 	mkdir -p vendor/github.com/magefile
 	cp -R ${BEAT_GOPATH}/src/github.com/elastic/beats/vendor/github.com/magefile/mage vendor/github.com/magefile
+	ls -la vendor/github.com/magefile
```

Comment:
Was this for testing?

Reply:
Yeah, I did a bunch of debugging to solve the case of the missing github.com/magefile/mage/build directory. It was the rsync exclude.

I'll fix this.